package.json

```diff
@@ -13,7 +13,15 @@
   "stripes": {
     "type": "settings",
     "displayName": "ui-myprofile.meta.title",
-    "route": "/myprofile"
+    "route": "/myprofile",
+    "links": {
+      "userDropdown": [
+        {
+          "caption" : "ui-myprofile.caption",
```

Comment:
@MikeTaylor it is okay. One note here is about a path ` "userDropdown.ui-myprofile.changePassword.caption"`. As I understand that requires us to create the following folders structure in the `translation`

```
|_userDropdown 
   |_ ui-myprofile
       |_ en.json
```
in addition to the existing ones

```
|_ ui-myprofile
    |_ en.json
```
which seems a bit overhead to me. Could we use just the following one 
` "ui-myprofile.userDropdown.changePassword.caption"`?

Reply:
@MikeTaylor that means we misunderstood you. Based on the ``` "caption" : "userDropdown.ui-myprofile.changePassword.caption",``` from your previous comment we pictured the following structure
```
|_userDropdown 
   |_ ui-myprofile
       |_ en.json
``` 
but now it seems that in en.json  under ui-myprofile will be enough to have `ui-myprofile.userDropdown.changePassword.caption` like in following picutre
![image](https://user-images.githubusercontent.com/37537790/43211851-31f46cc6-903b-11e8-826f-424d2bf22f30.png)
Am I right?

Reply:
I think so.

... though part of me thinks that the translation will need to live as part of stripes-core instead, which would be a bit weird.

Reply:
For me it seems quite logical to leave translations in ui-myprofille because we declare links in this module, so it is ok that this module is responsible for providing translations for those links.  If this contradicts with you please notify me and I will move translations to stripes-core module.

Reply:
Sorry, I was unclear. I agree that _conceptually_ the translations belong in ui-myprofile. We shouldn't move them if we can avoid it.

What I am a little concerned about is that they may not then be accessible from stripes-core, which is where they need to be accessed from in order to populate the captions in the user dropdown menu. But on reflection, it should be OK, since it has access to the module-name.

Reply:
I checked and it works.

Reply:
Great!

Reply:
@MikeTaylor chaneged